background.js

```diff
@@ -31,18 +32,21 @@ function resetState(tabId) {
   };
 }
 
-function bufferData(tabId, message) {
-  var tabData = data[tabId],
-      scope;
-
+function setIsHintFlag(message) {
```

Comment:
For my own clarification, what does this accomplish over having the code in `bufferData`?